cep-20.2.md

```diff
@@ -0,0 +1,832 @@
+
+  # would love to get rid of this:
+  # merge the build and host environments (used in many R packages on Windows)
+  # was `merge_build_host`
+  merge_build_and_host_envs: bool (defaults to false)
+
+  # include files even if they are already in the environment
+  # as part of some other host dependency
+  always_include_files: [path]
+
+  # do not soft- or hard-link these files, but always copy them was `no_link`
+  always_copy_files: [glob]
+
+  variant:
+    # Keys to forcibly use for the variant computation (even if they are not in the dependencies)
+    use_keys: [string]
+
+    # Keys to forcibly ignore for the variant computation (even if they are in the dependencies)
+    ignore_keys: [string]
+
+    # used to prefer this variant less
+    # note: was `track_features`
+    down_prioritize_variant: integer (negative, defaults to 0)
+
+  # settings concerning only Python
+  python:
+    # List of strings, where each string follows this format:
+    # PythonEntryPoint: `bsdiff4 = bsdiff4.cli:main_bsdiff4`
+    entry_points: [PythonEntryPoint]
+
+    # Specifies if python.app should be used as the entrypoint on macOS
+    # was `osx_is_app`
+    use_python_app_entrypoint: bool (defaults to false)  # macOS only!
+
+    # used on conda-forge, still needed?
+    preserve_egg_dir: bool (default false)
+
+    # skip compiling pyc for some files (was `skip_compile_pyc`)
+    skip_pyc_compilation: [glob]
+
+  # settings concerning the prefix detection in files
+  prefix_detection:
+    # force the file type of the given files to be TEXT or BINARY
+    # for prefix replacement
+    force_file_type:
+      # force TEXT file type
+      # was `???`
+      text: [glob]
+      # force binary file type
+      # was `???`
+      binary: [glob]
+
+    # ignore all or specific files for prefix replacement
+    # was `ignore_prefix_files`
+    ignore: bool | [path] (defaults to false)
+
+    # wether to detect binary files with prefix or not
+    # was `detect_binary_files_with_prefix`
+    ignore_binary_files: bool (defaults to true on Unix and (always) false on Windows)
+
+  # settings for shared libraries and executables
+  dynamic_linking:
+    # linux only, list of rpaths (was rpath)
+    rpaths: [path] (defaults to ['lib/'])
+
+    # wether to relocate binaries or not. If this is a list of paths, then
+    # only the listed paths are relocated
+    binary_relocation: bool (defaults to true) | [glob]
+
+    # Allow linking against libraries that are not in the run requirements
+    # (was `missing_dso_whitelist`)
+    missing_dso_allowlist: [glob]
+
+    # Allow runpath / rpath to point to these locations outside of the environment
+    # (was `runpath_whitelist`)
+    rpath_allowlist: [glob]
+
+    # what to do when detecting overdepending
+    overdepending_behavior: OneOf<"ignore" | "error"> # (defaults to "error")
+
+    # what to do when detecting overlinking
+    overlinking_behavior: OneOf<"ignore" | "error"> # (defaults to "error")
```

Comment:
Will `ignore` also print a warning or literally omit the report? Or the calculation? Maybe we need three options:

- skip
- warn
- error